src/helpers/sendGoogleNotification.ts

```diff
@@ -3,23 +3,25 @@ import firebase from '@/helpers/firebase'
 
 const messaging = getMessaging(firebase)
 
-function createNotification(title?: string) {
+function createNotification(title?: string, body?: string) {
   const baseNotification = {
     data: {
       type: 'newPost',
     },
   } as MessagingPayload
   if (title) {
     baseNotification.notification = {
-      title,
+      title: title,
```

Comment:
Don't duplicate `title: title`, you can use `title,` as shortcut, it automatically does `title: title`